packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -207,7 +197,7 @@ const CompleteResetPassword = ({
          * We may also want to consider putting a different error message in place for when
          * PW reset succeeds, but one of these fails. At the moment, the try/catch in Account
          * just returns false for these if the request fails. */
-        const [sessionIsVerified, hasTotp] = await Promise.all([
+        const [sessionIsVerified] = await Promise.all([
           account.isSessionVerifiedAuthClient(),
           account.hasTotpAuthClient(),
```

Comment:
I think we can remove this `Promise.all` with this and just do `sessionIsVerified = isSessionVerifiedAuthClient()`?